src/index.js

```diff
@@ -29,11 +29,6 @@ export default function supernova(env) {
           senseNavigation,
         });
       },
-      resize() {},
-      willUnmount() {
-        teardown(this.element);
-      },
-      destroy() {},
     },
```

Comment:
@miralemd are these functions mandatory? I tried in sense and in nebula but seems to work just the same without these

Reply:
you don't need the methods if they're not doing anything